tests/test_read.py

```diff
@@ -539,12 +539,12 @@ def test_read_solution_multiple_vehicle_types(ok_small_multi_depot):
     assert_equal(routes[1].vehicle_type(), 1)
 
 
-def test_multi_trip_instance():
+def test_multi_trip_instance(ok_small_multiple_trips):
     """
     Tests that a small multi-trip instance with vehicle reload options is
     parsed correctly.
     """
-    data = read("data/OkSmallMultipleTrips.txt")
+    data = ok_small_multiple_trips
```

Comment:
I didn't use a fixture here, because the fixture is essentially what we're testing (does `read()` work?).